src/Mapbender/CoreBundle/Resources/public/mapbender.element.redlining.js

```diff
@@ -29,10 +30,11 @@
         },
         _setup: function(){
             this.map = $('#' + this.options.target).data('mapbenderMbMap').map.olMap;
+            this.model = new Mapbender.Model();
```

Comment:
Why create a new instance? You are not creating your entire own layer stack, you cooperate with the main map.  
Use `this.map.model` to access the same model instance the main map uses.